gremlin-javascript/example/package.json

```
@@ -0,0 +1,14 @@
+{
+  "name": "example",
+  "version": "1.0.0",
+  "description": "GLV example for JavaScript with Node",
+  "main": "example.js",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1"
+  },
+  "author": "",
+  "license": "ISC",
```

```suggestion
  "license": "Apache-2.0",
```
Any source code contributed to an Apache project needs to be licensed under the Apache 2.0 license